Simplenote/AccountRemote.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+// MARK: - AccountVerificationRemote
+//
+class AccountRemote: Remote {
+    // MARK: Performing tasks
+
+    /// Send verification request for specified email address
+    ///
+    func verify(email: String, completion: @escaping (_ result: Result) -> Void) {
```

Comment:
Since we're changing things around, let's switch to Swift's [Result Type](https://developer.apple.com/documentation/swift/result) here